ecl2df/summary.py

```diff
@@ -295,10 +302,9 @@ def df(
         # Warning is already logged by eclfiles.
         return pd.DataFrame()
 
-    dframe = eclsum.pandas_frame(time_index_arg, column_keys)
+    # dframe = eclsum.pandas_frame(time_index_arg, column_keys)
```

Comment:
Remove?
```suggestion
```

Reply:
I kept these to signify that I want to roll back to using libecl for this.